entrypoint.sh

```diff
@@ -4,6 +4,11 @@
 # CLI arguments
 PY_VERSIONS=$1
 BUILD_REQUIREMENTS=$2
+SYSTEM_REQUIREMENTS=$3
+
+if [ ! -z "$SYSTEM_REQUIREMENTS" ]; then
+    yum install -y ${SYSTEM_REQUIREMENTS}
+fi
```

Comment:
The option is called `system-packages` in action.yml. We should keep this naming consistent in the entrypoint.sh. `SYSTEM_REQUIREMENTS` should be changed to `SYSTEM_PACKAGES`.